client-spark/spark-3/src/main/scala/org/apache/spark/shuffle/celeborn/CelebornShuffleReader.scala

```diff
@@ -89,6 +92,23 @@ class CelebornShuffleReader[K, C](
       }
     }
 
+    val exceptionMaker = new ExceptionMaker() {
+      override def makeException(
+          appShuffleId: Int,
+          shuffleId: Int,
+          partitionId: Int,
+          e: Exception): Exception = {
+        new FetchFailedException(
+          null,
+          handle.shuffleId,
```

Comment:
This should be spark's shuffle id ?

```suggestion
          appShuffleId,
```